src/Tribe/Admin/Notice/Php_Version.php

```diff
@@ -92,18 +92,11 @@ public function should_display() {
 	 * @return string
 	 */
 	public function display_notice() {
-
-		// for PHP version above 5.4 and up to 5.6
-		if ( version_compare( phpversion(), '5.4.0' ) >= 0 ) {
-			$text = __( 'Starting March 2019, %1$s will no longer support versions prior to PHP 5.6. Your site is currently using PHP version %2$s which will no longer be supported by %1$s. For best results, we recommend using PHP 5.6 or above.', 'tribe-common' );
-		} else {
-			// for PHPversions below 5.4
-			$text = __( 'Starting March 2019, %1$s will no longer work with versions prior to PHP 5.4. Currently your site is using PHP version %2$s. For best results, we recommend using PHP 5.6 or above.', 'tribe-common' );
-		}
+		// PHP versions older than 7.4
+		$text = __( 'Starting February 2023, %1$s will no longer work with versions prior to PHP 7.4. Currently your site is using PHP version %2$s. For best results, we recommend using PHP 7.4 or above.', 'tribe-common' );
```

Comment:
Translation comment for placeholders?

Reply:
Something like 
`/* Translators: %1$s list of plugins, %2$s current PHP version. */`